src/aktualizr_secondary/secondary_tcp_server.cc

```diff
@@ -139,13 +140,20 @@ bool SecondaryTcpServer::HandleOneConnection(int socket) {
         resp->present(AKIpUptaneMes_PR_putMetaResp);
         auto r = resp->putMetaResp();
         r->result = ok ? AKInstallationResult_success : AKInstallationResult_failure;
+        if (ok) {
+          LOG_TRACE << "Image root :\n" << meta_pack.image_root;
+          LOG_TRACE << "Image targets :\n" << meta_pack.image_targets;
+          LOG_TRACE << "Image snapshot :\n" << meta_pack.image_snapshot;
+          LOG_TRACE << "Image timestamp :\n" << meta_pack.image_timestamp;
+        }
       } break;
       case AKIpUptaneMes_PR_sendFirmwareReq: {
         auto fw = msg->sendFirmwareReq();
         auto send_firmware_result = impl_.sendFirmware(ToString(fw->firmware));
         resp->present(AKIpUptaneMes_PR_sendFirmwareResp);
         auto r = resp->sendFirmwareResp();
         r->result = send_firmware_result ? AKInstallationResult_success : AKInstallationResult_failure;
+        LOG_TRACE << "Download : " << (send_firmware_result ? "sucess" : "failed");
```

Comment:
```suggestion
        LOG_INFO << "Download " << (send_firmware_result ? "successful" : "failed") << ".";
```
Can we also print something about the Target, like the name and version?